pyproject.toml

```diff
@@ -11,6 +11,8 @@ python = "^3.9"
 jishaku = "^2.5.1"
 aiohttp = "^3.8.3"
 typing-extensions = "^4.3.0"
+discord-ext-menus = {git = "https://github.com/Rapptz/discord-ext-menus"}
```

Comment:
Shouldnt this be locked to a specific commit?

Reply:
Good catch, I assumed Poetry would do it when I did `poetry install`, I will fix that ASAP!